README.md

```diff
@@ -120,6 +120,16 @@ For now, productionizing is a bit manual. The minimum steps are:
      extension](https://wicg.github.io/webpackage/draft-yasskin-httpbis-origin-signed-exchanges-impl.html#cross-origin-cert-req).
      One provider of SXG certs is [DigiCert](https://www.digicert.com/account/ietf/http-signed-exchange.php).
      You MUST use this in `amppkg.toml`, and MUST NOT use it in your frontend.
+  6. Restart amppkg every ~90 days with after renewing your SXG cert (this
```

Comment:
Rather than "~90" can you say "90 days or fewer". Precise language is helpful in understanding the requirement.